### DISCUSSION TITLE
Problem with Student Verification Document Submission

### DISCUSSION THREAD

**Discussion by jkotania** [community/community#128123] [GitHub Education]
### Hi everyone,

**Hello GitHub Community,**

I am currently facing an issue with submitting my student verification document. Despite following the provided instructions, I am unable to successfully upload my document for verification.

I am a student at the Silesian Academy, formerly known as Wyższa Szkoła Techniczna (Polish name). My institution recently underwent a name change, and I suspect this could be causing some confusion or issues with the verification process. The document I am trying to upload reflects the old name of the institution.

After uploading my ID I got 2 errors:
"The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation."
I cannot include multiple documents together, as we know you can just upload one picture at the time. My ID doesn't have logo on it, even if it had, it wouldn't be valid.

"Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information."
It does contains my name exactly as it appears on ID and in Billing information.

I have tried to upload my student ID/other document (whichever is applicable), but it seems there is a problem with the submission process. I have ensured that my document is clear, legible, and contains all the necessary information.

I would appreciate any assistance or guidance on how to resolve this issue. I am eager to access the GitHub Student Developer Pack and explore all the learning resources it offers.

**Thank you in advance for your help.**

**Comment by Areyes42** [comment]
Hi. It sounds like you're dealing with a couple of frustrating issues. Here’s how you can address each of them:

### 1\. **School Name Visibility:**

Given that your institution recently changed its name from Wyższa Szkoła Techniczna to Silesian Academy, it’s likely causing a mismatch in the verification process. You could try these options:

-   **Include a Letter from Your Institution:** Ask your school administration for an official letter or document that mentions both the old and new names. This letter should explain the name change and confirm your enrollment. Upload this along with your student ID.
    
-   **Combine Documents into a Single Image:** While the upload system only accepts one image at a time, you can use tools like Paint, GIMP, or any image editing software to combine your student ID and the letter into a single image file. This way, you can show both your ID and proof of the name change in one upload.
    

### 2\. **Name Match Issue:**

Ensure your name is consistent across all documents and your GitHub billing information. If you’ve already confirmed that your name matches, consider the following steps:

-   **Double-Check All Fields:** Even a small discrepancy (like middle names or initials) can cause issues. Make sure that every part of your name is exactly the same in your student document and GitHub billing information.
    
-   **Submit a Support Ticket:** If everything is correct and you’re still facing problems, you might need direct assistance from GitHub Support. Provide them with screenshots and detailed explanations of your issue.
    

### Additional Tips:

-   **Use High-Quality Images:** Ensure your document is clear and all text is readable. Sometimes, even minor blurriness can lead to rejection.
    
-   **Include a Note:** When uploading, you might be able to add a note explaining the name change. If not, a detailed description in your support ticket should help.
    

Following these steps should help you resolve the verification issues. Good luck with accessing the GitHub Student Developer Pack!

**Comment by samus-aran** [comment]
> I am a student at the Silesian Academy, formerly known as Wyższa Szkoła Techniczna (Polish name). My institution recently underwent a name change, and I suspect this could be causing some confusion or issues with the verification process. The document I am trying to upload reflects the old name of the institution.

Hi @jkotania I suspect this might be the issue as well.

- Do you have any other newer documentation with the new name of the school. Include that a live capture of your ID. Else you will need to submit a ticket to update the school details on our records.
- [The image you selected does not appear to contain your school name.](https://github.com/orgs/community/discussions/111352#image-does-not-appear-to-contain-your-school-name)

If you raise a support ticket please include evidence of the school's name change 👍 

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly.

**Comment by umershahid84** [comment]
I am having issues with verification too. First of all, there is no way to upload a file. They really want us to take a picture of the verification document. Secondly, I uploaded my ID but their system does not accept it at all. I am trying to upload my transcript but there is no way to upload it. I tried reaching out to them but no one ever responded. This is frustrating.
